lib/src/rules/use_setters_to_change_a_property.dart

```diff
@@ -62,7 +62,7 @@ class _Visitor extends SimpleAstVisitor {
         node.returnType?.type?.name != 'void') {
       return;
     }
-    bool _visitExpression(Expression expression) {
+    _visitExpression(Expression expression) {
```

Comment:
I don't have a strong opinion, but the absence of a return type means that the return type defaults to `dynamic`, which means that the method can return anything. If the lint should only lint methods that don't return a value, then linting when the return type is implicit seems wrong.